docs/sources/setup-grafana/image-rendering/_index.md

```diff
@@ -26,7 +26,7 @@ While an image is being rendered, the PNG image is temporarily written to the fi
 
 A background job runs every 10 minutes and removes temporary images. You can configure how long an image should be stored before being removed by configuring the [temp_data_lifetime]({{< relref "../configure-grafana#temp_data_lifetime" >}}) setting.
 
-You can also render a PNG by hovering over the panel to display the actions menu in the top right corner, and then clicking **Share > Direct link rendered image** in the Link tab.
+You can also render a PNG by hovering over the panel to display the actions menu in the top-right corner, and then clicking **Share > Share link**. When you share an internal panel link, you also have the option to share it as an image.
```

Comment:
The sentence starts talking about rendering a PNG. Does it make sense to say 
`When you share an internal panel link, you also have the option to share it as an image.`? 
Should it be something like `and then clicking **Share > Share link**. Render image .`

Reply:
I think I didn't phrase it well. Let me try again.
```suggestion
You can also render a PNG by hovering over the panel to display the actions menu in the top-right corner, and then clicking **Share > Share link**. The **Render image** option is displayed in the link settings.
```

Reply:
sounds perfect!